### PR TITLE
consolidations, withdrawals: reject callvalue when calldatasize == 0

### PR DESCRIPTION
**Pull request by fjl**

### CODE REVIEW

File: src/withdrawals/main.eas [hunk 1]
```diff
@@ -66,6 +65,12 @@
   iszero                ;; [calldatasize != 0]
   jumpi @check_input
 
+  ;; Reject any callvalue here to prevent lost funds.
```
Comment (mkalinin) [inline src/withdrawals/main.eas:68]: we shouldn’t do this as the callvalue is used to cover the requests fee further down the line

Comment (fjl) [reply src/withdrawals/main.eas:68]: Maybe it's confusing with the branching here, but this branch of the program is the 'read the current excess fee' operation.

Comment (mkalinin) [reply src/withdrawals/main.eas:68]: Ah, I see!

**Comment by fjl** [comment]
@lightclient are you still open to merging this?

**Review by lightclient — APPROVED** [review]
SGTM

**Pull request merged**